data/json/furniture_and_terrain/appliances.json

```diff
@@ -106,6 +106,17 @@
     ],
     "flags": [ "CIRCLE_LIGHT", "APPLIANCE", "ENABLED_DRAINS_EPOWER", "CTRL_ELECTRONIC" ]
   },
+  {
+    "id": "ap_standing_floodlight",
+    "type": "vehicle_part",
+    "name": { "str": "standing floodlight" },
+    "item": "standing_floodlight",
+    "location": "structure",
+    "description": "A very bright, circular light that illuminates the area around it when switched on.",
+    "flags": [ "CIRCLE_LIGHT", "ENABLED_DRAINS_EPOWER", "APPLIANCE", "CTRL_ELECTRONIC" ],
+    "folded_volume": "0 L",
```

Comment:
This is required as removing the FOLDABLE tag is not enough. Without this I still get the fold option which doesn't make sense for an appliance